README.rst

```diff
@@ -44,14 +44,16 @@ Why PyGMT?
 A beautiful map is worth a thousand words.
 To truly understand how powerful PyGMT is, play with it online on `Binder <https://github.com/GenericMappingTools/try-gmt>`__!
 But if you need some convincing first, watch this `1 hour introduction <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ to PyGMT!
+For a quicker introduction, check out our `3 minute overview <https://youtu.be/4iPnITXrxVU>`__!
```

Comment:
I think we can just keep the 3 min part here. Suggest moving the 1 hour intro sentence after the 3 min video.

Reply:
Perhaps we should remove the link to the 1-hour video, because we already have more update-to-date tutorials and gallery examples.

Reply:
I removed the tutorial link from this paragraph and added a reference to the external resources page in the following paragraph. Let me know if you rather only keep references to our internal gallery and examples.